homework1_aboutme_styles.css

```diff
@@ -0,0 +1,41 @@
+body {
+	font-family: helvetica;
+}
+
+nav {
+	font-size: 28px;
+	padding-top: 10px;
+	text-align: center;
+
+}
+
+h1 {
+	font-size: 40px;
+	text-align: center;
+}
+
+h3 {
+	font-size: 26px;
+	padding-left: 100px;
+	padding-right: 25px;
+	padding-top: 50px;
+}
+
+img {
+	margin-left: 20px;
```

Comment:
Great use of margin rules here.  I know we did not cover this in time for this particular homework assignment, but for next time, think about how you would combine these 4 rules into the 1 "shorthand" margin rule.  (Same for the padding rules above in your h3 selector.)  You will actually see the shorthand for margin and padding more often than not.